test/integration/repo-init/expected/core-services/prow/02_config/_config.yaml

```diff
@@ -45,49 +45,45 @@ tide:
   max_goroutines: 20
   queries:
   - includedBranches:
-    - openshift-4.1
```

Comment:
The changes on the global one should not happen happened, as expected: The config for the new repo is in its own folder.